components/clubList.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import React, { PureComponent } from 'react';
+import { FlatList, View } from 'react-native';
+import { List, ListItem, Text } from 'react-native-elements';
+import connect from '../redux/connect';
+import * as actions from '../redux/modules/clubs';
+import { getIsLoading, getClubs } from '../redux/selectors/clubs';
+import noop from '../utils/noop';
+
+
+@connect(
+  {
+    clubs: getClubs,
+    isLoading: getIsLoading
+  },
+  { load: actions.load }
+)
+class ClubList extends PureComponent {
+  static defaultProps = {
+    clubs: [],
+    isLoading: false,
+    load: noop
+  };
+
+  componentDidMount() {
+    this.props.load();
+  }
+
+  renderItem = ({ item }) => (
+    <ListItem
+      title={`${item.name}`}
+      rightTitle={`${item.points} Bodova`}
+      rightTitleStyle={{ color: '#ff5252' }}
+      subtitle={`Broj članova: ${item.racers_count}`}
+    />
+  );
+
+  render() {
+    const { clubs, load, isLoading } = this.props;
+
+    return (
+      <View>
+        <List {...{ style }}>
+          <FlatList
+            keyExtractor={({ id }) => id}
+            renderItem={this.renderItem}
+            data={clubs}
+            ListEmptyComponent={<Text> Ucitavam klubove </Text>}
```

Comment:
I think something saying that there's no clubs would be better description for the app state. Saying that loading is going on might fool users into waiting for something that might not happen. What are your thoughts on that?